cookbooks/aws-parallelcluster-slurm/templates/default/slurm/head_node/update_munge_key.sh.erb

```diff
@@ -15,6 +15,21 @@ MUNGE_USER="<%= @munge_user %>"
 MUNGE_GROUP="<%= @munge_group %>"
 SHARED_DIRECTORY_COMPUTE="<%= @shared_directory_compute %>"
 SHARED_DIRECTORY_LOGIN="<%= @shared_directory_login %>"
+CHECK_LOGIN_NODES_SCRIPT_PATH="<%= node['cluster']['scripts_dir'] %>/slurm/check_login_nodes_status.sh"
+
+# Create a temporary file to store the output
+TEMP_FILE=$(mktemp)
```

Comment:
You can use the `trap` command to ensure that the file is removed once the script exits.
```
trap "rm -f ${TEMP_FILE}" EXIT
```

Reply:
Have deleted this logic. Instead, now we are using the exit code to check if LoginNodes are stopped.